examples/demo2/app.rs

```diff
@@ -82,10 +103,142 @@ impl App {
             KeyCode::Down | KeyCode::Char('j') => {
                 context.row_index = context.row_index.saturating_add(1);
             }
+            KeyCode::Char('d') => {
+                self.mode = Mode::Destroy;
+            }
             _ => {}
         };
         Ok(())
     }
+
+    fn should_quit(&self) -> bool {
+        self.mode == Mode::Quit
+    }
+}
+
+/// delay the start of the animation so it doesn't start immediately
+const DELAY: usize = 240;
+/// higher is means more pixels per frame are modified in the animation
```

Comment:
```suggestion
/// higher means more pixels per frame are modified in the animation
```

Reply:
Addressd in https://github.com/ratatui-org/ratatui/pull/809/commits/5337d0b74d3861624f16d685b7decd8c1b4f2397